### ISSUE TITLE
Performance on testing always better than validation

### ISSUE THREAD

**Issue by eugenelet** [hushell/sib_meta_learn#2]
Hi there, an observation that I can't comprehend is that the performance on test set is always better than on the validation set. I'm running on CIFAR-FS 1-shot using ConvNet for example. Isn't the sampling process for both the same or I'm missing something out? Thanks!

Edit: this only applies to Cifar-fs, the accuracy on miniImageNet is intuitive.

**Comment by hushell** [comment]
Hello, sorry for the late reply! 

We also observed the same phenomenon. It's probably because the validation set was created with more hard examples by chance. The sampling processes are the same in val and test, but all episodes in val are fixed during training. We didn't bother by this issue since the higher val-acc implies higher test-acc in general. Hope this answer helps.

**Comment by eugenelet** [comment]
Hello Shell! It's okay, we appreciate that you take time to look into this issue and share your thoughts. 

Yes, that's what I'm thinking as well after going through the code several times. Since the test-set has more classes it should be more complicated by intuition. Perhaps it's just a coincidence that the input distribution contained in the test-set is much simple when compared to the val-set. Thanks again for your time. :)

**Comment by hushell** [comment]
Glad it helps :)

Thank you for your feedback!
